.github/workflows/cut_release_branch.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           if [[ $NEXT_RELEASE =~ ([0-9]+\.[0-9]+\.[0-9]+) ]]; then
             echo "NEXT_VERSION_IN_BASE_BRANCH=${BASH_REMATCH[1]}" >> $GITHUB_ENV
           else
-            echo "The input for NEXT_RELEASE does not match a valid format [0-9]+\.[0-9]+\.[0-9]+"
+            echo "The input for NEXT_RELEASE does not match a valid format [0-9]+\.[0-9]+"
```

Comment:
I think this one should remain unchanged, right?

Reply:
oops, let me fix that. my initial plan was to update the "next release" regex too but realized that was going to complicate what got piped to the version update on master, so I went back and reverted that change. must have missed it